EasyLearn/app/helpers/login_helper.rb

```diff
@@ -0,0 +1,2 @@
+module LoginHelper
+end
```

Comment:
If you don't use this file, you can delete it.

Reply:
I agree with Yil.

Reply:
File deleted